src/matlab.ts

```diff
@@ -8,28 +8,31 @@ import * as fs from "fs";
 import properties from "./properties.json";
 import * as script from "./script";
 
-export interface Version {
-    release: string;
-    semantic: string;
+export interface Release {
+    name: string;
+    version: string;
+    updateNumber: string;
```

Comment:
Should this be an int? Is it a string because of "latest"? Maybe -1 represents latest?

Reply:
Maybe just name the field "update" then?